iceoryx2-cal/src/communication_channel/message_queue.rs

```diff
@@ -161,22 +161,22 @@ impl Default for Configuration {
 }
 
 impl NamedConceptConfiguration for Configuration {
-    fn prefix(mut self, value: FileName) -> Self {
-        self.prefix = value;
+    fn prefix(mut self, value: &FileName) -> Self {
+        self.prefix = *value;
```

Comment:
Hehe, move is ugly for the C bindings. It's a pity but the performance impact should be negligible